src/index.html

```diff
@@ -124,6 +124,9 @@ <h1 id="gameTitle">Pokéclicker</h1>
 
         <!--Middle column-->
         <div id="middle-column" class="col-lg-6 col-md-12 order-md-first">
+            <!-- Achievement Tracker -->
+            @import "achievementTrackerContainer.html"
```

Comment:
I tried this as both the current "bar at the top of the screen" and as a box in the right sidebar similar to the Quests list - I thought the approach here looked better, but happy to adjust if it's too intrusive.

FWIW, here was my pass at the sidebar box: 
<img width="289" alt="Screen Shot 2020-09-30 at 11 54 18 PM" src="https://user-images.githubusercontent.com/2694761/95019907-0f528900-0636-11eb-84e4-65f451a70265.png">